### DISCUSSION TITLE
ValueError: Attempting to unscale FP16 gradients.

### DISCUSSION THREAD

**Discussion by MilesQLi** [LAION-AI/Open-Assistant#3355] [Q&A] [answered]
The command is 
~/repos/Open-Assistant/model/model_training$ python trainer_sft.py --configs galactica-125m webgpt_dataset_only
 
Python 3.10, libraries installed as instructed.

[error_log.txt](https://github.com/LAION-AI/Open-Assistant/files/11537180/error_log.txt)

**Comment by jeangeezhang** [comment]
I got same issue
Python == 3.10
peft == 0.3.0
torch == 1.13.1

**Comment by andreaskoepf** [comment, marked as answer]
Please try adding `dtype: float32` to the model configuration.

**Comment by theophilegervet** [comment]
@MilesQLi @jeangeezhang I'm facing the same issue. Were you able to fix it?

**Comment by bhauptvogel** [comment]
Is it not possible to train the model with lower precision? e.g. `dtype: float16`
